gridstatus/utils.py

```diff
@@ -345,3 +345,16 @@ def move_cols_to_front(df, cols_to_move):
     for c in cols_to_move:
         cols.remove(c)
     return df[cols_to_move + cols]
+
+
+def check_latest_value_time(df: pd.DataFrame, file: str):
+    """Check the latest value time in the DataFrame"""
+    value_col_map = {
```

Comment:
i think it's more maintainable to add another parameter to `_get_historical` that is the column to use for this calculation.

we would then require all the callers of `_get_historical` (e.g `_get_historical_fuel_mix`, `_get_historical_load`) to provide the column they want to use. 

by doing this, if we were to add another endpoint like https://www.caiso.com/outlook/sp/history/20240929/rtm_forecast.csv, we wouldn't have to touch any of these lower level util functions